### ISSUE TITLE
build_transformer_model()传入的参数application

### ISSUE THREAD

**Issue by hjing100** [bojone/bert4keras#296]
build_transformer_model()传入的参数application可以是encoder、lm、unilm，我看到你2020.3.5更新application=seq2seq为application=unilm，所以application=unilm是默认使用unilm进行seq2seq类型的任务吗？因为unilm是可以进行NLU任务的。

提问时请尽可能提供如下信息：

### 基本信息
- 你使用的**操作系统**: 
- 你使用的**Python**版本: python3.6
- 你使用的**Tensorflow**版本: 1.14
- 你使用的**Keras**版本: 2.3.1
- 你使用的**bert4keras**版本: 0.9.7
- 你使用纯**keras**还是**tf.keras**: 
- 你加载的**预训练模型**:

### 核心代码
```python
# 请在此处贴上你的核心代码。
# 请尽量只保留关键部分，不要无脑贴全部代码。
```

### 输出信息
```shell
# 请在此处贴上你的调试输出
```

### 自我尝试
不管什么问题，请先尝试自行解决，“万般努力”之下仍然无法解决再来提问。此处请贴上你的努力过程。

**Comment by i4never** [comment]
？叫``seq2seq``时候用的就不是unilm了吗？

**Comment by bojone** [comment]
这里的unilm是指做seq2seq时的一种attention mask，并不是一个特定的模型。
